test/import.js

```diff
@@ -155,14 +154,16 @@ test("should work with no styles without throwing an error", t => {
 
 test("should be able to consume modules in the custom-resolve way", t => {
   const resolve = require("resolve")
-  const sassResolve = (file, opts) => {
-    opts = opts || {}
-    opts.extensions = [ ".scss", ".css" ]
-    opts.packageFilter = pkg => {
-      pkg.main = pkg.sass || pkg.style || "index"
-      return pkg
-    }
-    return resolve.sync(file, opts)
+  const sassResolve = (id, base, opts) => {
```

Comment:
if the signature of the callback to provide custom resolve, we should also document that and specify this in the CHANGELOG.